### PR TITLE
Adding WAF Rule Exclusion endpoints to go-fastly client

### PR DESCRIPTION
**Pull request by matpimenta**

### CODE REVIEW

File: fastly/waf_rule_exclusion.go [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+	// The Web Application Firewall's version number.
+	WAFVersionNumber int
+	// Limit results to exclusions with the specified exclusions type.
+	FilterExclusionType *string
+	// Limit results to exclusions with the specified exclusion name.
+	FilterName *string
+	// Limit results to exclusions that represent the specified ModSecurity modsec_rule_id.
+	FilterModSedID *string
+	// Include relationships. Optional, comma-separated values. Permitted values: waf_rule_revision and waf_firewall_version.
+	Include *string
```
Comment (phamann) [inline fastly/waf_rule_exclusion.go:76]: Same as above, this should be a `[]string`.

**Review by phamann — CHANGES_REQUESTED** [review]
Looking good. One comment/change around an input API change from `string` to `[]sting`.

File: fastly/waf_rule_exclusion.go [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+	// Limit results to exclusions with the specified exclusion name.
+	FilterName *string
+	// Limit results to exclusions that represent the specified ModSecurity modsec_rule_id.
+	FilterModSedID *string
+	// Limit the number of returned pages.
+	PageSize *int
+	// Request a specific page of exclusions.
+	PageNumber *int
+	// Include relationships. Optional, comma-separated values. Permitted values: waf_rule_revision and waf_firewall_version.
+	Include *string
```
[the remark this answers is not in the document]
Comment (matpimenta) [reply fastly/waf_rule_exclusion.go]: Yes, it definitely looks nicer with a slice. I've updated the PR with this change

File: fastly/waf_rule_exclusion.go (hunk 1 above)
Comment (matpimenta) [reply fastly/waf_rule_exclusion.go]: Updated to slice as well

**Review by phamann — APPROVED** [review]
👍 LGTM

**Pull request merged**